### DISCUSSION TITLE
v3.000 RC2

### DISCUSSION THREAD

**Discussion by kenorb** [EA31337/EA31337#398] [Releases]
New EA31337 v3.000 RC2 Pre-release.

Changelog:

- Introduction of Meta strategies.
- Over 30 of new [Meta strategies](https://github.com/EA31337/EA31337-strategies-meta).
- New MA strategies: MA_Breakout, MA_Cross_Pivot, MA_Cross_Shift, MA_Cross_Sup_Res, MA_Cross_Timeframe, MA_Trend.
- New Oscillator strategies: Oscillator, Oscillator_Cross, Oscillator_Cross_Shift, Oscillator_Cross_Timeframe, Oscillator_Cross_Zero, Oscillator_Divergence, Oscillator_Martingale, Oscillator_Multi, Oscillator_Overlay, Oscillator_Range, Oscillator_Trend.
- New Elite version which introduces complex configurations by layering multiple strategies on top of each other based on certain market conditions.
- **Full Changelog**: https://github.com/EA31337/EA31337/compare/v2.013.1...v3.000

<hr /><em>This discussion was created from the release <a href='https://github.com/EA31337/EA31337/releases/tag/v3.000-rc1'>v3.000 RC1</a>.</em>

**Comment by brubrant** [comment]
What exactly are the META strategies? What are the difference from the regular strategies?

**Reply by kenorb** [reply]
Meta strategies they're like strategies of other strategies.
Regular strategy relies on indicator as trading signals,
meta strategy relies on other strategies as signals.

**Comment by PeshawaEng** [comment]
how can I use this to XAUUSD? what period and how to edit the parameters?

**Comment by groovystag-cell** [comment]
I have loaded successfully the latest version 3.000 on the MT5 trader but it is not taking any trades or doing anything.
